src/microbe_stage/Membrane.cs

```diff
@@ -542,6 +542,8 @@ private void BuildMesh()
         float multiplier = 2.0f * Mathf.Pi;
         var center = new Vector2(0.5f, 0.5f);
 
+        movementWigglyNess = Type.MovementFactor;
```

Comment:
This is wrong, the wigglyness shouldn't be set from the movement factor. Movement factor is only how much the base movement speed is modified.
A new property should be added to Type that is the movementwigglyness value and is set from the json to make things configurable.